### PR TITLE
fix: OPTIC-1248: Fixed error message that pops up when user checks heidi notifications checkbox at account settings 

### PR DESCRIPTION
**Pull request by mcanu**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

### Describe the reason for change
We were sending _email_ as part of the newsletters subscription request, which raised an error [here](https://github.com/heartexlabs/label-studio/blob/f4c3185bf3f1b50ed62edd91e328f4e1cd56a17f/label_studio/users/api.py#L186-L190)

#### What does this fix?
Error messaging popping up when clicking on _Get the latest news & tips from Heidi_ checkbox.

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: label_studio/users/templates/users/user_account.html [hunk 1]
```diff
@@ -233,7 +233,6 @@
       <table>
         <tr><td style="{% if user.allow_newsletters is None %}border: 1px red solid; border-radius: 5px{% endif %}">
 
-          <input name="email" type="hidden" value="{{ user.email }}">
           <input name="first_name" type="hidden" value="{{ user.first_name }}">
           <input name="last_name" type="hidden" value="{{ user.last_name }}">
```
Comment (luarmr) [inline label_studio/users/templates/users/user_account.html]: Nice find, wondering if we need to send first_name and last_name also.

**Review by luarmr — APPROVED** [review]

File: label_studio/users/templates/users/user_account.html (hunk 1 above)
Comment (mcanu) [reply label_studio/users/templates/users/user_account.html]: I don't think they are necessary either, but we will be taking this page to React this sprint, so we'll remove this pretty soon.

Comment (bmartel) [reply label_studio/users/templates/users/user_account.html]: That was my thoughts, its a PATCH to the API so typically speaking no. It should only send what is changing.

**Review by bmartel — APPROVED** [review]

File: label_studio/users/templates/users/user_account.html (hunk 1 above)
Comment (mcanu) [reply label_studio/users/templates/users/user_account.html]: agreed, I'll remove them

**Pull request merged**